CoFrancePlugIn.cpp

```diff
@@ -58,6 +58,9 @@ CoFrancePlugIn::CoFrancePlugIn(void):CPlugIn(EuroScopePlugIn::COMPATIBILITY_CODE
     RegisterTagItemType("Stand", CoFranceTags::STAND);
     RegisterTagItemFunction("Stand popup", CoFranceTags::FUNCTION_STAND_MENU);
 
+    RegisterTagItemType("Assigned runway (colored)", CoFranceTags::COLORED_RWY);
```

Comment:
The tag name is "assigned runway" but seems to related only to departure runway or am I reading this wrong?

Reply:
There is apparently only one tag name related to runway which is used for both departure and arrival runway:

<img width="505" alt="image" src="https://github.com/user-attachments/assets/25f4f4cc-20c1-40f9-9a09-8f5a2e1d2f4a" />

It is defined as:
```
const   int     TAG_ITEM_TYPE_ASSIGNED_RUNWAY           = 47 ;
```